docs/books/user-guide/using-console.adoc

```diff
@@ -17,110 +17,100 @@ specific language governing permissions and limitations
 under the License
 ////
 
+[id='using-router-console']
 = Using {ConsoleName}
 
-[[console-overview]]
-== Console Overview
+{ConsoleName} is a web console for monitoring the status and performance of {RouterName} router networks.
 
-The console is an HTML based web site that displays information about a qpid dispatch router network.
+== Setting up access to the web console
 
-The console requires an HTML web server that can serve static html, javascript, style sheets, and images.
+Before you can access the web console, you must configure a `listener` to accept HTTP connections for the web console and serve the console files.
 
-The console only provides limited information about the clients that are attached to the router network and is therfore more appropriate for administrators needing to know the layout and health of the router network.
+.Procedure
 
-[[console-installation]]
-== Console Installation
+. On the router from which you want to access the web console, open the {RouterConfigFile} configuration file.
```

Comment:
{RouterConfigFile} is unresolved in the HTML. Is this a new alias?

Reply:
Yes, the attribute is unresolved because it was introduced in another PR that hasn't been merged yet.